src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    /* Is it this class has a next logic? One of the class fields must be File entity.
+    Because in work situation in eighteen's line of code we need to pass File entity in constructor.
+    After that we are going to call writeToFile() method. WorkWithFile class is kind of
+    utility class.
+    */
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] splitInformationFromFile = readFromFile(fromFileName);
+        String report = createReport(splitInformationFromFile);
+        WorkWithFile file = new WorkWithFile();
+        file.writeToFile(toFileName, report);
+    }
```

Comment:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String[] splitInformationFromFile = readFromFile(fromFileName);
        String report = createReport(splitInformationFromFile);
        writeToFile(toFileName, report);
    }
```
this method the same as `createReport` and `readFromFile` why do you create WorkWithFile instance to invoke method